.storybook/main.ts

```diff
@@ -14,6 +14,9 @@ const config: StorybookConfig = {
         options: {},
     },
     staticDirs: ['../public'],
+    typescript: {
```

Comment:
something odd was happening here — all of a sudden I couldn't view this story, and the error came from storybook's react docgen code (nothing from our side changed or looks wrong). Disabling it fixes it. I don't think we even use this feature so it probably doesn't matter. I imagine it is just a bug that will eventually be fixed on their side.